crates/uv-cli/src/lib.rs

```diff
@@ -154,6 +154,10 @@ pub struct GlobalArgs {
     /// Show the resolved settings for the current command.
     #[arg(global = true, long, hide = true)]
     pub show_settings: bool,
+
+    /// Hides all progress bars when set
+    #[arg(global = true, long, conflicts_with_all = ["quiet", "verbose"])]
```

Comment:
Why conflict with quiet? Seems fine to request silence twice without erroring :)

Reply:
Ah yes, it makes sense. I thought since it conflicts with `--verbose` it should also conflict with `--quiet`. 
I'll change that. 👍

Reply:
I removed both. `--verbose` doesn't show progress indicators already so it doesn't conflict, and `--quiet` doesn't show anything. 😁